ChaosMod/Effects/db/Meta/MetaReInvoke.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+	Effect By OnlyRealNubs
+*/
+
+#include "stdafx.h"
+
+static void OnStart()
+{
+	EffectDispatcher* disp = GetComponent<EffectDispatcher>();
+	std::vector<RegisteredEffect*> eff = disp->GetRecentEffects(5);
```

Comment:
Should accept a const reference here (copy elision should kick in here but still)